mariadb-database-id/3.initialize-database.sql

```diff
@@ -0,0 +1,20 @@
+USE eventuate;
+
+CREATE TABLE new_message (
```

Comment:
Isn't this in common?!

Reply:
Yes, but the same approach is used for testing json schema. Probably is better to move such sqls to java code or gradle plugin, I did not think about it yet. What do you think?

Reply:
No. My point is: why is the CDC building images that have already been built in eventuate-common.